### DISCUSSION TITLE
"VersionConflict: (spacy 2.2.3 (/opt/conda/lib/python3.6/site-packages), Requirement.parse('spacy==3.1.2'))

### DISCUSSION THREAD

**Discussion by pratikchhapolika** [explosion/spaCy#9058] [Help: Installation] [answered]
## Your Environment

* Operating System: **Linux/ posix**
* Python Version Used: **Python 3.6.12 |Anaconda, Inc.|**
* spaCy Version Used: **Spacy 3.1.2**
* Environment Information: **Anaconda**

I ran this command's:

```
pip uninstall spacy==2.2.3
pip install spacy
```

When I do:

`spacy.__version__`

**2.2.3**

I do not understand, even after successfully  uninstalling spacy 2.2.3 I get this error.

I tried this:
```

import pkg_resources
pkg_resources.require("spacy==3.1.2")
import spacy
```
```
---------------------------------------------------------------------------
VersionConflict                           Traceback (most recent call last)
/usr/local/bin/kernel-launchers/python/scripts/launch_ipykernel.py in <module>
      1 import pkg_resources
----> 2 pkg_resources.require("spacy==3.1.2")
      3 import spacy

~/.local/lib/python3.6/site-packages/pkg_resources/__init__.py in require(self, *requirements)
    884         included, even if they were already activated in this working set.
    885         """
--> 886         needed = self.resolve(parse_requirements(requirements))
    887 
    888         for dist in needed:

~/.local/lib/python3.6/site-packages/pkg_resources/__init__.py in resolve(self, requirements, env, installer, replace_conflicting, extras)
    775                 # Oops, the "best" so far conflicts with a dependency
    776                 dependent_req = required_by[req]
--> 777                 raise VersionConflict(dist, req).with_context(dependent_req)
    778 
    779             # push the new requirements onto the stack

VersionConflict: (spacy 2.2.3 (/opt/conda/lib/python3.6/site-packages), Requirement.parse('spacy==3.1.2'))


```

**How can I completely wipe out spacy old version and download 3.1.2**?

**Comment by polm** [comment, marked as answer]
I think this is a weird thing conda does where it tries to avoid upgrading packages you already have installed. Try creating a new blank env and installing spaCy in that and you should get the latest version. Try this:

```
conda create --name myenv --no-default-packages
```

And then follow [the usual install instructions](https://spacy.io/usage).

**Reply by pratikchhapolika** [reply]
> I think this is a weird thing conda does where it tries to avoid upgrading packages you already have installed. Try creating a new blank env and installing spaCy in that and you should get the latest version. Try this:
> 
> ```
> conda create --name myenv --no-default-packages
> ```
> 
> And then follow [the usual install instructions](https://spacy.io/usage).

I am using Jupyter server which is not controlled by me. So I cannot create new envs.

**Reply by polm** [reply]
If you have that little control over your environment I'm sorry but whatever you do will be difficult. Can you ask the admin of the Jupyter server to create a new env for you?
